ext/storage_sources/s3_store/test/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+project(s3_store_tests CXX)    
+include(${CMAKE_SOURCE_DIR}/test/ctest_helpers.cmake)
+
+# Build the unit test binary.
+create_test_executable(run_s3_unit_tests 
+    SOURCES test_aws_bucket_conn.cxx
+    LIBS aws-sdk::s3-crt aws-sdk::core aws_bucket_util
+    INCLUDES ${CMAKE_CURRENT_SOURCE_DIR} ${AWSSDK_INCLUDE_DIRS}
```

Comment:
Apologies, one last thing I missed. This line can be deleted,`${AWSSDK_INCLUDE_DIRS}` doesn't exist & `${CMAKE_CURRENT_SOURCE_DIR}` isn't needed since there are no includes in the `ext/storage_sources/s3_store/test` directory.

Reply:
I removed the INCLUDES - can the libs declaration stay?

Reply:
Yep, the `LIBS` are still good. You don't need to remove them.